### DISCUSSION TITLE
Repo settings in the GitHub mobile app

### DISCUSSION THREAD

**Discussion by okyanusoz** [community/community#38409] [Mobile]
I would like to add collaborators and change the settings of repo in the GitHub mobile app.

At the time of writing, this is not possible.

I would appreciate it if you could add repo settings to your mobile app.

**Comment by MarcusSebushi** [comment]
> I would like to add collaborators and change the settings of repo in the GitHub mobile app.
> 
> 
> 
> At the time of writing, this is not possible.
> 
> 
> 
> I would appreciate it if you could add repo settings to your mobile app.

**Comment by LoftyDroid** [comment]
read this: https://docs.github.com/en/get-started/using-github/github-mobile

**Comment by jfullstackdev** [comment]
if the feature is missing, you can always log in using the regular browser of your mobile to experience the full version as a workaround for now while it's not yet there

**Reply by ryanalexmartin** [reply]
Well... the app hijacks GitHub links, so you'd have to uninstall the app every time you want to use the website version. Not fun.

**Reply by michael-harley** [reply]
The mobile web redirects using deeplink, into the mobile app.
This is not possible.

Tried with the desktop site view, but still not working. Definitely not a fun experience if I need to uninstall the app every single time.

**Comment by Molnj** [comment]
I could open the repo within the browser by opening the link in a new private tab.
